acme/dask_helpers.py

```diff
@@ -319,14 +344,15 @@ def esi_cluster_setup(partition="8GBXS", n_jobs=2, mem_per_job="auto", n_jobs_st
 
     # Create `SLURMCluster` object using provided parameters
     out_files = os.path.join(slurm_wdir, "slurm-%j.out")
+    job_extra.append("--output={}".format(out_files))
```

Comment:
Very pedantic comment: I think it might be worthwhile checking if the user passed its own `--output` parameter before we force-set it here. I can implement this real quick, if you want :)

Reply:
Hey Katharine - I just incorporated the `--output` parsing mentioned above. I'll go ahead and merge into dev (not main yet, I want to include some other fixes before releasing a new version). Thank you for your work getting this of the ground!